src/sentry/static/sentry/app/views/settings/projectGeneralSettings.jsx

```diff
@@ -384,7 +386,11 @@ class ProjectGeneralSettings extends AsyncView {
             jsonFormProps.features.has('tweak-grouping-config')) && (
             <JsonForm
               {...jsonFormProps}
-              title={t('Grouping Settings')}
+              title={
+                <>
```

Comment:
We've been using `<React.Fragment>` over `<>` generally as React.Fragment is simpler to understand.

Reply:
@jan-auer take note!